test/sapphire_devnet.test.ts

```diff
@@ -240,6 +240,37 @@ describe("torus utils sapphire devnet", function () {
     });
   });
 
+  it("should should fetch public address with keyType", async function () {
```

Comment:
can you use existing test case and user for testcase to validate nothing changes on passing keyType for existing test user

Reply:
`Willa_Funk11@gmail.com` is existing testcase's email in the sapphire_devnet_ed25519.test.ts